imagersite/imager_profile/handlers.py

```diff
@@ -0,0 +1,54 @@
+from .models import ImagerProfile
+from django.contrib.auth.models import User
+from django.dispatch import receiver
+from django.db.models.signals import post_save, post_delete
+
+
+@receiver(post_save, sender=User)
+def create_user_profile(sender, **kwargs):
+    """A User should always have an ImagerProfile"""
+    instance = kwargs.get('instance')
+    if not instance or kwargs.get('raw', False):
+        return
+    try:
+        instance.profile
+    except ImagerProfile.DoesNotExist:
+        instance.profile = ImagerProfile()
+        instance.profile.save()
+
+
+@receiver(post_save, sender=ImagerProfile)
+def create_user(sender, **kwargs):
+    """An ImagerProfile should always have a User"""
+    instance = kwargs.get('instance')
+    if not instance or kwargs.get('raw', False):
+        return
+    try:
+        instance.user
+    except User.DoesNotExist:
+        instance.user = User()
+        instance.user.save()
```

Comment:
There's a bit of a problem in doing this.  What should the user's username be?  Their password?  Their email address?  None of these things is clear and since all are required, it's probably better not to try to auto-create a User without them.